docs/_articles/en/faq.md

```diff
@@ -16,6 +16,7 @@ lang: en
 - [How do I _remove_ a language from `sfdx-scanner`?](/en/faq/#q-how-do-i-remove-a-language-from-sfdx-scanner)
 
 #### [Questions about adding/removing rules](/en/faq/#questions-about-addingremoving-rules-1)
+- [How do I enable Engine X's default rules for Language Y?](/en/faq/#q-how-do-i-enable-engine-xs-default-rules-for-language-y)
```

Comment:
The doc writer suggested taking out TOC as they are difficult to maintain it is the norm in salesforce to avoid it. She has made a change in her PR. You might hit a merge conflict here.

Reply:
What is TOC?

Reply:
Table of Contents